assetsTransformer.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path');
+
+module.exports = {
+    process(src, filename, config, options) {
```

Comment:
nit: IF you will add underscore to function parameter's names that are not in use - > eslint will not warn about them. (in case if you want to leave them named)
in this case: `process(src, filename, _config, _options) {`

Reply:
Pushed the fix.